docs/kubernetes/user-guides/snapshots.md

```diff
@@ -122,3 +125,150 @@
     lost+found
     sample-file.txt
     ```
+
+### Manually Provisioning Snapshot Contents.
```

Comment:
What do you think of calling it something like "Import a pre-existing snapshot"? I think this description aligns better with what a user would want to accomplish without using Kubernetes terms.

Reply:
Done